src/Store/StoreThread.re

```diff
@@ -244,29 +261,22 @@ let start =
       }
     );
 
+  // TODO: These should all be replaced with isolinear subscriptions.
   let _: Isolinear.Stream.unsubscribeFunc =
     Isolinear.Stream.connect(dispatch, inputStream);
   let _: Isolinear.Stream.unsubscribeFunc =
     Isolinear.Stream.connect(dispatch, vimStream);
   let _: Isolinear.Stream.unsubscribeFunc =
     Isolinear.Stream.connect(dispatch, editorEventStream);
-  let _: Isolinear.Stream.unsubscribeFunc =
-    Isolinear.Stream.connect(dispatch, syntaxStream);
   let _: Isolinear.Stream.unsubscribeFunc =
     Isolinear.Stream.connect(dispatch, extHostStream);
-  let _: Isolinear.Stream.unsubscribeFunc =
```

Comment:
There were several cases where we weren't using the stream at all (or it could be replaced by using an Effect-with-dispatch). I removed these to remove noise and make it easier to refactor later without touching all of these pieces.

The remaining items that are using the stream functionality could/should be replaced with subscriptions.

Reply:
> I don't think streams are really necessary at all. 

Agreed! 

> For an easier immediate simplification though, it might be an idea to just use a single stream, and pass its dispatch function into the stores instead of having the stores all return their own stream.

Sounds good.